src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -1,10 +1,12 @@
 package seedu.address.logic.commands;
 
+
 import static java.util.Objects.requireNonNull;
 
 import seedu.address.commons.core.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.FindPredicate;
+
 
 /**
  * Finds and lists all persons in address book whose name contains any of the argument keywords.
```

Comment:
I think you can change this line to all of the argument keywords to differentiate between this and findOr :)

Reply:
changed! thanks :)